04_numeronym/mohitnag/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	ns := make([]string, len(vals))
+	for i, val := range vals {
+		ns[i] = numeronym(val)
+	}
+	return ns
+}
+
+func numeronym(v string) string {
+	rs := []rune(v)
```

Comment:
You shouldn't need to create another `rs` variable so that you have a `[]rune`. You can access the individual runes from the `[]string`. Please remove this and directly access from `v`

Reply:
Ryan, i think i need to keep the `[]rune`  variable to support unicode.

Reply:
True. I just tested this, I was under the impression that this still supported unicode. Thanks for pointing this out!